Production_scripts/Process_extracts/extract_a&e.R

```diff
@@ -134,7 +134,9 @@ ae_clean <- ae_file %>%
   # Create month variable
   mutate(month = strftime(record_keydate1, "%m")) %>%
   # Allocate the costs to the correct month
-  create_day_episode_costs(record_keydate1, cost_total_net)
+  create_day_episode_costs(record_keydate1, cost_total_net) %>%
+  # Add SMR type
+  mutate(smrtype = add_smr_type(recid))
```

Comment:
```suggestion
  create_day_episode_costs(record_keydate1, cost_total_net),
  # Add SMR type
  smrtype = add_smr_type(recid)
  )
```
Since there is a mutate above, we can wrap this together.

Reply:
Wrapped